src/common/components/elements/AppHeader.tsx

```diff
@@ -76,127 +73,32 @@ export function AppHeader() {
+            <DiscoverMenu />
 
-                  <Link
-                    href="https://docs.google.com/document/d/12uQU7LbLUd0bY7Nz13-F9cua5Wk8mnRNBlyDzF6gRmo"
-                    passHref
-                  >
-                    <a className="hover:underline" target="_blank">
-                      Privacy policy
-                    </a>
-                  </Link>
-                </div>
-              }
-            >
-              <a className="flex min-w-fit items-center">
-                About <ChevronRight color="#fff" className="ml-2 rotate-90 " />{' '}
-              </a>
-            </Popover> */}
-            {/* <Popover
-              placement="bottom"
-              content={
-                <div className="flex flex-col space-y-6">
-                  <Link
-                    href="https://holaplex-support.zendesk.com/hc/en-us/sections/4407417107475-FAQ"
-                    passHref
-                  >
-                    <a target="_blank" className="hover:underline">
-                      FAQ
-                    </a>
-                  </Link>
-                  <Link
-                    href="https://holaplex-support.zendesk.com/hc/en-us/sections/4407782141971-Set-Up-A-Store"
-                    passHref
-                  >
-                    <a target="_blank" className="hover:underline">
-                      Setting up a store
-                    </a>
-                  </Link>
-
-                  <Link
-                    href="https://holaplex-support.zendesk.com/hc/en-us/sections/4407791450515-Minting-NFTs"
-                    passHref
-                  >
-                    <a target="_blank" className="hover:underline">
-                      Minting NFTS
-                    </a>
-                  </Link>
-                  <Link
-                    href="https://holaplex-support.zendesk.com/hc/en-us/sections/4407792008979-Selling-NFTs"
-                    passHref
-                  >
-                    <a target="_blank" className="hover:underline">
-                      Selling NFTS
-                    </a>
-                  </Link>
-                  <Link href="https://holaplex-support.zendesk.com/hc/en-us/requests/new" passHref>
-                    <a target="_blank" className="hover:underline">
-                      Submit a support ticket
-                    </a>
-                  </Link>
-                </div>
-              }
-            >
-              <a className="flex min-w-fit items-center">
-                Help <ChevronRight color="#fff" className="ml-2 rotate-90 " />{' '}
-              </a>
-            </Popover> */}
-            {connectedAndInstalledWallet && <DialectNotificationsButton />}
-
-            {connectedAndInstalledWallet ? (
-              <ProfileImage />
-            ) : (
+            {connectedAndInstalledWallet && (
+              <>
+                <DialectNotificationsButton />
+                <ProfileImage />
```

Comment:
![image](https://user-images.githubusercontent.com/10433154/172837151-101ab559-1e83-4489-b106-cfde0aa02d6d.png)

Profile image does not load (top right), though this could be unrelated to the PR, please check if it works with connect, disconnect and reload :)

Reply:
This looks to be caused by the new wallet provider. Ideas?

> Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?
> 
> Check the render method of `Link`.
>     at Button5 (webpack-internal:///./src/common/components/elements/Button2.tsx:84:24)
>     at Link (webpack-internal:///./node_modules/next/dist/client/link.js:115:23)
>     at div
>     at nav
>     at AppHeader (webpack-internal:///./src/common/components/elements/AppHeader.tsx:86:72)

Reply:
Got the fix. It was already affecting production, so the fix already went in.